Create 25.05.0.md (#347)
o3de/sig-release: releases/25.05.0/25.05.0.md

  * Fixed TrackView example Python script  
* Script Canvas:  
  * Fix crash on Script Canvas editor close  
  * Update slot names if the referenced variable is renamed  
  * Fix default file path when saving a new graph  
  * Update the variable's property tree on rename  
  * Fixed issue with promoting variables from a node's slot  
  * Update slot names if the referenced variable is renamed  
  * Fixed deleted, or changed variables being left present in a graph’s variable map which led to many nondeterministic behaviors  
  * Fixed crash when opening invalid ScriptCanvas file  
  * Fixed graph deactivation error “Failed to get user data from graph. Constructed with invalid values”  
* Refactoring the MovieSystem so that it may be moved into a gem  
* Asset Processor analysis regarding how many assets are new files, outdated, have altered dependencies, or have new builders.

# Sig-Simulation

* SimulationInterfaces Gem: new Gem implementing standard ROS 2 simulation interfaces  
* ROS2 Gem: added segmentation feature to LiDAR sensor  
* ROS2 Gem: added notification bus to the spawner  
* ROS2 Gem: reimplemented PID controller  

> * ROS2 Gem: fixed bugs multiple bugs in ROS2Clock and joint manipulator; removed not working Python tests  

Reviewer: ```suggestion
* ROS2 Gem: multiple bugs in ROS2Clock and joint manipulator; removed not working Python tests  
```

  * Refactor Vulkan RenderPassBuilder to use a RenderAttachmentLayout  
  * Enable specialization constants for all RHI on all platforms  
  * Add support for subpasses to Vulkan and Metal  
* **Function/Specialization shader constants** \- Add support for specialization constants for shader options. When using specialization constants there's no need to create a shader variant, the values for shader options are baked at runtime. Also added batching of shader variants as part of the work for using specialization constants. Supported on DX12, Vulkan and Metal.  
* **GeometryView** \- Introduced a new class called GeometryView that helps to de-duplicate data between drawItems and drawPackets, as these now hold a pointer to the respective GeometryView, making both DrawPackets and DrawItems smaller in size and thus more cache friendly. Most importantly, it allows index and stream buffers and buffer views to be modified at runtime without having to rebuild DrawPackets/DrawItems.   

> * **Vertex color support \-** Add vertex color support to the BasePBR, StandardPBR and EnhancedPBR materials. Vertex color can be combined with the base color using a factor and a blending method that can be configured in the material. Vertex color can be enable/disable from the material

Reviewer 2: ```suggestion
* **Vertex color support** \- Add vertex color support to the BasePBR, StandardPBR and EnhancedPBR materials. Vertex color can be combined with the base color using a factor and a blending method that can be configured in the material. Vertex color can be enable/disable from the material
```
Fix bold formatting

## Minor features**

* Implement "Debug Ray Tracing" component \- Adds the level component "Debug Ray Tracing" which can be used to debug various properties of the ray tracing scene  
* Enable interoperability between rendering backends and other APIs  

> * Add Pass::ChangeConnection \- Adds a new method: Pass::ChangeConnection. Used to change how an Input or InputOutput slot is connected to a slot from a different pass.  

Reviewer 2: ```suggestion
* Add a new method: Pass::ChangeConnection for changing how an Input or InputOutput slot is connected to another slot from a different pass.  
```
Remove duplication

* Remove static library targets from Atom\_RPI  
* Add Blas compaction for raytracing  
* Added Compute Shader support in \*.MaterialPipeline  
* Add Indirect Dispatch and updated Fullscreen Dispatch for RPI::ComputePass and RPI::RayTracingPass  
* Add support for one RayTracingAccelerationStructurePass per device   
* Limit number of worker threads to capacity of m\_workerThreads   
* Extend Pass functionality to improve inserting passes.  
* Updating Vulkan Memory Allocator to 3.1.0 release  
* Add memory heaps for tracking Atom assets  
* Add ScopeQueries to CopyPass  
* Introduce scene/viewsrg\_all.srgi as a wrapper around scene/viewsrg.srgi  
* Improve Vulkan barriers  

> * Non-compositional Universal Scene Description geometry and animation support  

Reviewer 2: ```suggestion
* Add support for non-compositional Universal Scene Description (USD) geometry and animation 
```
Add acronym for simplicity

* Add a mutex to the RHIRequirmentRequest Ebus  
* Fixed race condition related with Material::m\_shaderVariantReadyEvent  
* Fix usage of RayTracing feature mask  
* Clang 19 fixes  
* Vulkan RHI: HDR Support  
* Fix a spurious crash with AttachmentReadback  
* ImGuiGpuProfiler support to ignore idle pipelines  
* Fixes Material Previews are not showing actual material  
* Fix some pass attachment binding-related functions not being updated to the new deque-based storage method

# AR/VR/XR

* OpenXRVk Gem: OpenXR dependency was updated (1.0.22 \-\> 1.1.41)  
* Improved the installation process for the Meta Quest Mobile SDK for O3DE. Installation is automatic now.

# Sig-docs-community

* Docs now make use of Hugo’s module system for the API docs, as part of the initiative to make docs easier to contribute to. Gone are the days of giant PRs in the o3de.org repository to update the API docs.  
* To that end, there have been some changes to the API docs generation process. Core contributors take note of the documentation for the new process in the sig-docs repo.  
* SIG-Simulation’s new work includes documentation updates for it. 🥳

# Known Issues in 25.05

> * O3DE may crash when creating projects using some reserved keywords as the project name (e.g. “project”). This will be addressed in a future point release.  

Reviewer 2: ```suggestion
```
Move to `sig-content` section

  
# Sig-Core

* RAM reduction by removing legacy asset table  
* Resolved issues with latest stable version of CMake

  
# Sig-Networking

* Fix player spawning when client connects to a server during a server level load  
* AWS cloud service gems moved to separate repo to keep engine agnostic. See [aws/o3de-repo](https://github.com/aws/o3de-repo)  
* Non-monolithic release mode servers supported  
* Fix spawning network prefabs containing more than one entity  
* Multiplayer component autogen throws compile time error if accidentally creating multiplayer components with the same name  
* Game session “level” property added to automatically loading levels when a multiplayer game session begins  
* Better checks and warnings around invalid IP addresses

# Sig-Platform

* Addressed the Python shared library conflict on Linux, which caused segmentation faults when using the ROS2 environment and Ros2 GEM with O3DE.  
* Disabled Android's APK compression, allowing PAK files to load without significant delays or softlocks.  
* Fixed Python script execution errors in the Shader Management Console on Linux, allowing shader variants to be created successfully.  

> * Proposed an RFC to add curated and uncurated lists of remote repositories to the project manager, improving discoverability.  

Reviewer 3: Hyperlink to the rfc if any

Author: Actually I will delete that line as it doesn't affect the release at all.
